<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtConceptMapViewTestsDialog</class>
 <widget class="QDialog" name="QtConceptMapViewTestsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View tests</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_display">
     <property name="text">
      <string>Display</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_edit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_rate">
     <property name="text">
      <string>Rate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea_display">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="display_contents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>255</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QScrollArea" name="scrollArea_edit">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="edit_contents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>254</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QScrollArea" name="scrollArea_rate">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="rate_contents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>255</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
